scripts/utilities/getPackagePath/index.js

```diff
@@ -4,5 +4,13 @@ module.exports = packageName => {
   const matchesPath = packagePath =>
     new RegExp(`/${packageName}$`).test(packagePath);
 
-  return getPackages().find(matchesPath);
+  const match = getPackages().find(matchesPath);
+
+  // covers case in jenkins where the base level `psammead` folder isnt
+  // simply called `psammead` but instead has a hash on the end
+  if(!match && packageName == 'psammead') {
```

Comment:
```suggestion
  if (!match && packageName === 'psammead') {
```

CI complaining about

Reply:
Done :)